src/Log/SentryLogger.php

```diff
@@ -332,4 +333,80 @@ public static function backtrace(array $record) : array
         ]);
     }
 
+    /**
+     * Get the information about the current release, if possible
+     * @return array
+     */
+    protected function getGitInfo()
+    {
+        // Initialise empty array to return
+        $return = [];
+        // If the file operations error out, we need to catch it
+        try {
+            $return = $this->getGitCommitMessage($return);
+        } catch (\Exception $exception) {
+            // Default message. As it's an extra, it's not overly crowding the interface
+            $return['Message'] = 'No git repo found or inaccessible';
+        }
+
+        return $return;
+    }
+
+    /**
+     * Get the HEAD tag or the release
+     *
+     * @return string
+     */
+    public static function getGitHead()
+    {
+        $head = file_exists(Director::baseFolder() . '/.git/HEAD');
+        if ($head) {
+            $head = explode(':', file_get_contents(Director::baseFolder() . '/.git/HEAD'));
```

Comment:
You've already made a F/S call using `file_get_contents()` to build your `$head` var, why not re-use it and avoid another?

```
if ($head = file_exists(Director::baseFolder() . '/.git/HEAD')) {
    // Do stuff with $head
    ...
}
```